mexico-city/src/mc_nitro.rs

```diff
@@ -0,0 +1,219 @@
+//! AWS Nitro-Enclaves-specific material for the Mexico City enclave
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use byteorder::{ByteOrder, LittleEndian};
+use nix::sys::socket::listen as listen_vsock;
+use nix::sys::socket::{accept, bind, recv, send, MsgFlags, SockAddr};
+use nix::sys::socket::{socket, AddressFamily, SockFlag, SockType};
+use std::os::unix::io::RawFd;
+use veracruz_utils::{MCMessage, vsocket, NitroRootEnclaveMessage, NitroStatus, receive_buffer, send_buffer};
+use std::os::unix::io::AsRawFd;
+use std::convert::TryInto;
+use nsm_io;
+use nsm_lib;
+
+const CID: u32 = 0xFFFFFFFF; // VMADDR_CID_ANY
```

Comment:
Can you add `rustdoc`-style comments to all of these constants, explaining what they are and where the values come from?  There's an open issue to try to add `rustdoc` comments throughout the existing codebase, with some changesets already merged adding them, and we hopefully should get to a point soon where the entire codebase is properly commented.